appserver/extras/grpc/pom.xml

```diff
@@ -49,14 +49,14 @@
     <artifactId>grpc</artifactId>
     <name>Payara gRPC Support</name>
     <packaging>jar</packaging>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
 
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <grpc.version>1.24.0</grpc.version>
+        <grpc.version>1.43.1</grpc.version>
```

Comment:
Is this correct?

Reply:
It _exists_, so if we're going to use it yes we should update the name of the issue.

Reply:
I already update the ticket and also the titles of the PR's